src/app/(with-header)/mypage/profile/page.tsx

```
@@ -0,0 +1,182 @@
+          <label className='mb-16 block text-[24px] leading-[32px] font-bold text-black'>
+            비밀번호
+          </label>
+          <Input
+            type='password'
+            value={formData.newPassword}
+            onChange={handleInputChange('newPassword')}
+            onBlur={handlePasswordBlur}
+            error={errors.newPassword}
+            placeholder='8자 이상 입력해 주세요'
+          />
+        </div>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)